docs/get-started.md

```diff
@@ -24,9 +24,12 @@ If you have not signed up before, you can do so [here](https://improbable.io/get
 
 - Make sure you download the **Installer** version, and select the following components during installation:
     - **Linux Build Support**
-    - **Mac Build Support**
+    - **Mac Build Support** 
 
-> **Note:** You need Linux build support because all server-workers in a cloud deployment run in a Linux environment. 
+> **Note:** 
+You need these build supports even though you are developing on a Windows PC.<br/>
+You need **Linux Build Support** because all server-workers in a cloud deployment run in a Linux environment. <br/>
+You need **Mac Build Support** because your end-users may play your game on a Mac, so SpatialOS requires it.
```

Comment:
that's not really true. they can disable building it for mac / windows if they want